fix compile warnings on gcc-4.x
Joerg Bornkessel <hd_brummy@gentoo.org> 14 Nov 2007
diff -Naur dvd-cvs-20071113.orig/player-dvd.c dvd-cvs-20071113/player-dvd.c
--- dvd-cvs-20071113.orig/player-dvd.c	2007-11-13 17:57:34.000000000 +0100
+++ dvd-cvs-20071113/player-dvd.c	2007-11-13 17:57:58.000000000 +0100
@@ -2768,7 +2768,7 @@
 void cDvdPlayer::GetAudioLanguageStr(const char **AudioLanguageStr) const
 {
 	static char buffer[100];
-	char *audioTypeDescr = NULL;
+	const char *audioTypeDescr = NULL;
 
     if (!DVDActiveAndRunning()) {
         *AudioLanguageStr = "n.a.";
